PWGHF/D2H/Tasks/taskCharmPolarisation.cxx

```diff
@@ -520,10 +548,66 @@ struct TaskPolarisationCharmHadrons {
           registry.add("hRecoNonPromptRandom", "THn for polarisation studies with cosThStar w.r.t. random axis -- reco non-prompt signal", HistType::kTHnSparseF, {thnAxisInvMass, thnAxisPt, thnAxisNumPvContributors, thnAxisY, thnAxisInvMassKPiLc, thnAxisCosThetaStarRandom, thnAxisResoChannelLc, thnAxisAbsEtaTrackMin, thnAxisNumItsClsMin, thnAxisNumTpcClsMin, thnAxisIsRotatedCandidate, thnAxisCharge});
         }
       }
+    } else if (doprocessDstarInPbPb || doprocessDstarMcInPbPb) {
```

Comment:
It would be good to preserve all the pp options also in Pb-Pb, but I agree that we should remove the axis for the rotational background to reduce the size. You could consider to define a `vector` of `AxisSpec` and add the axis for the rotational background only if it is not Pb-Pb, similarly to this:
https://github.com/AliceO2Group/O2Physics/blob/d7e0aab14e2fea3ced2893851fee3577b557c612/PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx#L175-L190
Moreover if EP is activated, you should throw a `FATAL` error